src/test/java/com/rakovets/course/java/core/practice/exception_handling/CustomArithmeticExceptionTest.java

```diff
@@ -0,0 +1,31 @@
+package com.rakovets.course.java.core.practice.exception_handling;
+
+import com.rakovets.course.java.core.practice.exception_handling.exception.CustomArithmeticException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class CustomArithmeticExceptionTest {
+    CustomArithmeticExceptionTask task = new CustomArithmeticExceptionTask();
+    CustomArithmeticExceptionDemo demo = new CustomArithmeticExceptionDemo();
+
+    @Test
+    public void testProduceCustomArithmeticExceptionZero() {
+        Assertions.assertThrows(CustomArithmeticException.class, () -> task.produceCustomArithmeticException(2, 0));
```

Comment:
catch a specific exception

Reply:
I didn't understand exactly essence of correction. Could you explain,please